pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -354,7 +376,42 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard {
         );
 
         _managementSwapFeePercentage = managementSwapFeePercentage;
-        emit ManagementFeePercentageChanged(managementSwapFeePercentage);
+        emit ManagementSwapFeePercentageChanged(managementSwapFeePercentage);
+    }
+
+    /**
+     * @notice Sets the yearly percentage AUM management fee which is payable to the pool manager.
+     * @dev Attempting to collect AUM fees in excesss of 10% will result in this function reverting.
+     */
+    function setManagementAumFeePercentage(uint256 managementAumFeePercentage) external authenticate whenNotPaused {
+        // We want to avoid a pool manager being able to retroactively increase the amount of AUM fees payable.
+        // We then perform a collection before updating the fee percentage to prevent this.
+        collectAumManagementFees();
+
+        _setManagementAumFeePercentage(managementAumFeePercentage);
+    }
+
+    function _setManagementAumFeePercentage(uint256 managementAumFeePercentage) private {
+        _require(
+            managementAumFeePercentage <= _MAX_MANAGEMENT_AUM_FEE_PERCENTAGE,
+            Errors.MAX_MANAGEMENT_AUM_FEE_PERCENTAGE
+        );
+
+        _managementAumFeePercentage = managementAumFeePercentage;
+        emit ManagementAumFeePercentageChanged(managementAumFeePercentage);
+    }
+
+    /**
+     * @notice Collect any accrued AUM fees and send them to the pool manager.
+     * @dev This can be called by anyone to collect accrued AUM fees - and will be called automatically on
+     * joins and exits.
+     */
+    function collectAumManagementFees() public whenNotPaused nonReentrant {
```

Comment:
```suggestion
    function collectAumManagementFees() public whenNotPaused {
```

We could technically remove this modifier as we don't make any external calls in this function.